Performance/QPSBenchmark/Sources/QPSBenchmark/grpc-swift-config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "invocations": [
+    {
+      "protoFiles": [
+        "Model/benchmark_service.proto",
+        "Model/worker_service.proto",
+      ],
+      "visibility": "public"
+    }
+  ],
+  "protocPath": "/usr/local/bin/protoc"
```

Comment:
I don't think we should define the path here. Did you do this for local or CI builds?

Reply:
Ah, didn't realise about this. I did it so it built locally, because otherwise it fails. CI has passed, I wonder if it would pass without the `protocPath` defined here?

Reply:
It works fine locally launching with `xed`. I've removed the path - let's see if the CI fails. If it does, I still don't have write access to the repo (have asked @glbrntt) so I won't be able to edit the actions.

Reply:
I think the CI is going to work since it doesn't test the QPS package build. Furthermore, you can edit the actions by editing the yml in the `.github` folder at the root of the repo.

Reply:
Indeed, it has passed :)